libgraphics/include/pivot/graphics/types/Transform.hxx

```diff
@@ -38,5 +61,23 @@ public:
 
     /// Scale component
     glm::vec3 scale = glm::vec3(1.0f);
+
+    /// Root of the transform
+    pivot::EntityRef root = EntityRef::empty();
 };
+
+/// Ostream overload for glm::vec3
+inline std::ostream &operator<<(std::ostream &os, const glm::vec3 &vec)
+{
+    return os << "(" << vec.x << "," << vec.y << "," << vec.z << ")";
+}
```

Comment:
There is a function in glm to convert vec3 (or any glm object) to string, but if i remember well it does not compile in cpp20